packages/matchbox/src/components/Panel/SubHeader.js

```diff
@@ -17,7 +18,7 @@ const SubHeader = React.forwardRef(function SubHeader(props, userRef) {
       fontSize="200"
       fontWeight="normal"
       lineHeight="200"
-      color="gray.700"
+      color={appearanceContext === 'inverted' ? 'white' : 'gray.700'}
```

Comment:
SubHeader is typically a little muted, should we use a darker gray when its inverted?